Modified Docs as required in issue #223

YashJipkate opened the pull request.

EdgeKing810 commented:
Why did you leave so many blank lines? :/
And some lines have a looooot of identation at the start...

ppabcd commented on docs/build/html/general/freeradius.html:
No empty line at the end of a file.

ppabcd commented on docs/build/html/general/freeradius.html:
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+            <div class="wy-nav-content">
+
+                <div class="rst-content">
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+                <a href="../index.html">django-freeradius</a>
+
+            </nav>
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+
+        <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
+
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+                        <li class="toctree-l1"><a class="reference internal" href="contributing.html">Contributing</a></li>
+                        <li class="toctree-l1"><a class="reference internal" href="goals.html">Motivations and Goals</a></li>
+                    </ul>
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+                </div>
+
+                <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+                    <div class="version">
+                        0.1
+                    </div>
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+
+          
+          </a>
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+
+
+                    <a href="../index.html" class="icon icon-home"> django-freeradius
+          
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+        <nav data-toggle="wy-nav-shift" class="wy-nav-side">
+            <div class="wy-side-scroll">
+                <div class="wy-side-nav-search">
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+
+
+    <div class="wy-grid-for-nav">
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+</head>
+
+<body class="wy-body-for-nav">
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+    <link rel="search" title="Search" href="../search.html" />
+    <link rel="next" title="Management commands" href="management_commands.html" />
+    <link rel="prev" title="Available settings" href="settings.html" />
+
```
Too many spaces.

ppabcd commented on docs/build/html/general/freeradius.html:
```diff
@@ -0,0 +1,727 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Installation and configuration of Freeradius 3 &mdash; django-freeradius 0.1 alpha documentation</title>
+
```
Too many spaces.

EdgeKing810 commented:
I didn't know we (students) could review... :/

ppabcd commented:
@EdgeKing810 contributor only can review without approve or decline.

YashJipkate commented:
@ppabcd Actually those spaces were already present there in the repo. Although I have now removed them in my last push.
And I also added newline at end of the file as said by you.

ppabcd commented:
https://www.freeformatter.com/html-formatter.html maybe will help you and don't forget to add new line at end of file. @YashJipkate

YashJipkate commented:
@ppabcd used that online formatter of yours and pushed again.

marfgold1 commented:
Don't you want to squash the commits and keep the latest commit message?

dwang commented:
@YashJipkate 

I see that you're currently editing the built HTML file, but instead you should edit the source RST (reStructuredText) file. You can find it at [docs/source/general/freeradius.rst](https://github.com/openwisp/django-freeradius/blob/master/docs/source/general/freeradius.rst). There should not be any HTML files in the docs at the end. [Here](http://docutils.sourceforge.net/docs/user/rst/quickref.html) is a quick reference to the reStructuredText syntax.

As @marfgold1 just said, once you are done, you should squash your commits using the local `git` client and keep your latest commit message as it is correct. 👍
